avssync.go

```diff
@@ -163,7 +164,7 @@ func (a *AvsSync) tryNTimesUpdateStakesOfEntireOperatorSetForQuorum(quorum byte,
 		a.logger.Infof("Updating stakes of operators in quorum %d: %v", int(quorum), operators)
 		timeoutCtx, cancel = context.WithTimeout(context.Background(), a.writerTimeoutDuration)
 		defer cancel()
-		_, err = a.avsWriter.UpdateStakesOfEntireOperatorSetForQuorums(timeoutCtx, [][]common.Address{operators}, []byte{quorum})
+		_, err = a.avsWriter.UpdateStakesOfEntireOperatorSetForQuorums(timeoutCtx, [][]common.Address{operators}, []types.QuorumNum{types.QuorumNum(quorum)})
```

Comment:
nit: don't think you need the inner `types.QuorumNum`

Reply:
You do need the type conversion 😭

Reply:
hmm weird. oh also there's a type `types.QuorumNums` which will pretty print the quorums, as opposed to `[]types.QuorumNum` which encodes as a base64 unreadable thing.